mempool/mempool.go

```diff
@@ -406,11 +406,11 @@ func (mem *Mempool) ReapMaxBytesMaxGas(maxBytes int, maxGas int64) types.Txs {
 		memTx := e.Value.(*mempoolTx)
 		// Check total size requirement
 		// amino.UvarintSize is not used here because it won't be possible to reuse buf
-		aminoOverhead := binary.PutUvarint(buf[:], uint64(len(memTx.tx)))
-		if maxBytes > -1 && totalBytes+len(memTx.tx)+aminoOverhead > maxBytes {
+		aminoOverhead := int64(binary.PutUvarint(buf[:], uint64(len(memTx.tx))))
```

Comment:
Is having to do this everywhere a code smell for our code, or for Go itself, or for neither?

Reply:
for our code